src/service_args.py

```diff
@@ -123,8 +123,6 @@ def __customize_service_args(self):
             self.__origintrail()
         elif self.chain_name == 'asset-hub-rococo':
             self.__asset_hub_rococo()
-        elif self.chain_name in ['bifrost-kusama', 'bifrost-polkadot']:
-            self.__bifrost()
```

Comment:
Just checking, did you test deploying the charm with these chains after removing this? @gflow33

Reply:
Yes, I tested it locally and it worked